concrete_datastore/settings/base.py

```diff
@@ -174,10 +194,14 @@
 
 
 PASSWORD_CHANGE_TOKEN_EXPIRY_HOURS = 4
-
-SECURE_CONNECT_EXPIRY_TIME_DAYS = 2
+SECURE_CONNECT_TOKEN_EXPIRY_TIME_SECONDS = 2 * 3600 * 24  # 2 days
```

Comment:
I checked and the previous setting was not used on the current running instances so this change will be transparent